src/main/kotlin/uk/gov/dluhc/printapi/service/PrintRequestsService.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.dluhc.printapi.service
+
+import mu.KotlinLogging
+import org.springframework.beans.factory.annotation.Value
+import uk.gov.dluhc.printapi.database.entity.PrintDetails
+import uk.gov.dluhc.printapi.database.entity.Status
+import uk.gov.dluhc.printapi.database.repository.PrintDetailsRepository
+import uk.gov.dluhc.printapi.messaging.MessageQueue
+import uk.gov.dluhc.printapi.messaging.models.ProcessPrintRequestBatchMessage
+
+private val logger = KotlinLogging.logger { }
+
+class PrintRequestsService(
```

Comment:
Is this going to be annotated as a `@Service` and wired into the `BatchPrintRequestsJob`?

Reply:
Yeah I just pushed that actually